Day1/P1.py

```diff
@@ -0,0 +1,300 @@
+#string capitalize method: change first character in a string to upper case, while other characters remain lower case
+from gettext import find
+
+
+a = "hello, how are you?"
+print(a.capitalize())
+
+def capitalizestr(txt):
+    if txt[0] == 'a':
+        new_txt = 'A' + txt[1:]
+    elif txt[0] == 'b': #question: why elif but not if after the first if statement?
```

Comment:
Do you still have this question? Think about the difference between 2 consecutive `if` vs `if ... elif ....`